_projects/permaculture-design-tools.md

```diff
@@ -7,6 +7,6 @@ labels:
 - Business Tools
 - Open Data
 current_stage: Project Pitch
```

Comment:
Should there be a link to the pitch (that would be hosted somewhere on the website)?